pkg/subscriber/processor/evaluation_events_dwh.go

```diff
@@ -196,13 +196,19 @@ func (w *evalEvtWriter) convToEvaluationEvent(
 		// Tag is optional, so we insert none when is empty.
 		tag = "none"
 	}
+	var variationId string
+	if e.VariationId == "" {
+		// if the reason is default, the variation_id is empty
+		// so we insert "default" as the variation_id
+		variationId = "default"
+	}
```

Comment:
In the current logic, if the `e.VariationId` is not empty, the `variationId` will be empty when saving the event to BigQuery, causing errors.

```suggestion
	variationID := e.VariationId
	if variationID == "" {
		// When the default value is returned to the app in the SDK,
		// it creates a default evaluation event with an empty variation ID.
		// Because we don't have the variation ID we insert "default" as the variation_id
		variationID = "default"
	}
```

Reply:
Oh,thank you!
I made a mistake...